web/experiments/feature_view.py

```diff
@@ -0,0 +1,268 @@
+from __future__ import print_function
+import datetime
+import logging
+from collections import defaultdict
+
+import numpy as np
+import pandas as pd
+import seaborn as sns
+from matplotlib import pyplot as plt
+from sklearn.linear_model import SGDClassifier
+from sklearn.metrics import f1_score, make_scorer
+from sklearn.model_selection import LeaveOneOut, GridSearchCV
+from tqdm import tqdm
+
+
+def process_CSLB(feature_matrix_path, *embs):
+    df = pd.read_csv(feature_matrix_path, sep='\t', index_col=0)
+
+    t = df.transpose()
+    t.iloc[t.values > 1] = 1
+
+    iv = df.index.values.astype(str)
+
+    word_to_rm = set([])
+
+    # filter double meaning words
+    for cw in iv:
+        sw = str(cw)
+
+        ssw = sw.split('_(')
+        if len(ssw) >= 2:
+            word_to_rm.add(sw)
+        else:
+            # remove two-word names according to paper
+            # 'guinea_pig', 'rolls_royce'
+            ssw = sw.split('_')
+            if len(ssw) >= 2:
+                word_to_rm.add(sw)
+
+    word_to_rm = [str(x) for x in word_to_rm]
+
+    # drop words non aviable in embeddings corpora
+    concepts_str = [str(x) for x in t.columns]
```

Comment:
Add here a warning how many words were dropped (this is important for users that haven't realized that two word embeddings might not be directly comparable due to different corpora)

Reply:
Done